Update README.md (#2)
LauWien/ece-teaching: README.md

> # Documentation of the teaching learning material by the Energy, Climate, and Environment Program (ECE)

Reviewer: Suggest to make a bit shorter, so that it can fit in one line (I hope)
```suggestion
# Overview of the teaching & learning material by the ECE program
```

> Copyright (c) 2021 IIASA

Reviewer: suggest to add (in the same line)
```
; licensed under the [MIT license](LICENSE)

![License](https://img.shields.io/github/license/iiasa/ece-teaching)

This is a [Sphinx](http://sphinx-doc.org/) project for the documentation of the build of the teaching and learning website by the Energy, Climate, and Environment Program (ECE). The website can be found [here](https://teaching.ece.iiasa.ac.at/).

## Scope & Aim

> The scope and aim of the repository is to create a Sphinx documentation, which is the basis for the teaching and learning website. This repository should contain all docs, but not the data, of the teaching and learning material. 

Reviewer: The repository does not document the build, it *is* the build - better to use more direct language. Also, please use hard line-breaks at 88 characters.

Probably also good to have a short section here (before the technical instructions) about what kind of material is included in "learning and teaching" - workshops organized by ECE, university courses taught by ECE staff, tutorials, etc.

Third comment, the difference between docs and data is very subtle and might be easy to grasp for all potential contributors.

Maybe better:
> This repo contains pages with relevant information about teaching events, including links to relevant websites and the teaching material (e.g., pdf, pptx, recorded videos)
> Any such material should not be committed to this repository, but be stored in an appropriate location
> such as the IIASA publication repository (pure.iiasa.ac.at) or a user folder on ldom002.

Author: Thanks for your review!
Just to be sure regarding the hard line-breaks: The width of the text on the page should not move if the screen gets extended but rather stay at max 88 characters. E.g.:
```
This repo contains pages with relevant information about teaching events, including <br/>
links to relevant websites and the teaching material (e.g., pdf, pptx, recorded videos). <br/>
Any such material should not be committed to this repository, but be stored in an <br/>
appropriate location, such as the IIASA publication repository (pure.iiasa.ac.at) or a <br/>
user folder on ldom002.
```

Reviewer: Didn't mean "no hard breaks in the rendered page" (not `<br/>`), just have line breaks in the rst file.

## Instructions

### How to install dependencies and build locally

> 1. Install dependencies and start Sphinx [sphinx_rtd_theme](https://sphinx-rtd-theme.readthedocs.io/)

Reviewer: Why is there a link to the rtd-theme? If you want to mention it, maybe write a full sentence like: "The teaching pages use the Sphinx-RTD-Theme customized to follow the IIASA style & design guidelines.

Author: Ah sorry, yes this link is not the correct one.

      `pip install -r requirements.txt`
      
      `$ sphinx-quickstart`

2. Build from the command line. On Linux or macOS:

    `make html`

   On Windows:

    `.\make html`
 
 3. Open the rendered html page to see if everything fits:
 
    `_build/html/index.html`

> ### How to add new material

Reviewer: maybe better use the word content?

Author: What do you mean by the word content and/or where can I find it?

Reviewer: Material -> Content

This is more generic, in my opinion...

Author: Oh my :D I thought we're talking about the content of a Microsoft Word document. 
Definitely content makes more sense!

Reviewer: Yeah, I should have used hyphens...

Author: I shouldn't have thought so complicated ;)

> To add new material, fork the repository and either update the existing ReST file `index.rst` or, if you want to build a subpage, implement your ReST file on the top level and link it within the `index.rst` file. 

Reviewer: this is a long sentence, maybe a sub-bullet-list is easier to read?

Reviewer: Important to state that a user should fork, then create a new branch.

And after editing the files, the user should start a PR to iiasa/main.

Implementing a subpage is only recommended if your materials covers series of lectures, including video tutorial(s) and multiple other files. 

### How to write

Use the following references for writing your ReST file:

- [Quick reStructuredText](http://docutils.sourceforge.net/docs/user/rst/quickref.html) reference from docutils.
- [ReST cheat sheet](https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html) by Thomas Cokelaer.
- [Sphinx](http://www.sphinx-doc.org/) documentation, including reference on Sphinx-specific ReST syntax.

> As the scope of the repository states - please do not store your data in the repository itself. Good places to upload files such as videos or presentation slides might be http://pure.iiasa.ac.at/id/eprint/17286/ or https://data.ene.iiasa.ac.at/. Then simply link to the folder within your ReST file. 

Reviewer: The url data.ene.iiasa.ac.at should not be used any more, please always use ece instead. Also, this particular URL forwards to a page in the IIASA CMS, so this is probably not intelligible to most users...

E.g. linking a video:

```
.. raw:: html

	<video width="640" height="360" controls preload="none" controlsList="nodownload">

> 		<source src="Your file path"type="video/mp4">

Reviewer: ```suggestion
		<source src="Your file path" type="video/mp4">
```

# Overview of the teaching & learning material by the ECE program

> Copyright (c) 2021 IIASA; ![License](https://img.shields.io/github/license/iiasa/ece-teaching)

Reviewer: Suggest to have both badge and text `MIT License`
```suggestion
![License](https://img.shields.io/github/license/iiasa/ece-teaching)

Copyright (c) 2021 IIASA; published under the [MIT License](LICENSE)
```

> This is a [Sphinx](http://sphinx-doc.org/) project for the documentation of the 
> build of the teaching and learning website by the Energy, Climate, and 
> Environment Program (ECE). The website can be found [here](https://teaching.ece.iiasa.ac.at/).

Reviewer: ```suggestion
This is a [Sphinx](http://sphinx-doc.org/) project for the documentation of the
teaching and learning activities by the Energy, Climate, and Environment Program (ECE).
The website can be found [here](https://teaching.ece.iiasa.ac.at/).
```

The MIT license applies only to this site and not to the teaching and learning
content.

## Scope & Aim

This repository contains pages with relevant information about teaching events, 
including links to relevant websites and the teaching material (e.g., pdf, pptx, 
recorded videos). Any such material should not be committed to this repository, 
but be stored in an appropriate location such as the IIASA publication 

> repository (pure.iiasa.ac.at) or a folder on ldom002.

Reviewer: ```suggestion
repository (pure.iiasa.ac.at) or a personal folder,
e.g., https://data.ece.iiasa.ac.at/<user>
```
